modules/core/04-channel/keeper/packet_test.go

```diff
@@ -656,6 +656,31 @@ func (suite *KeeperTestSuite) TestAcknowledgePacket() {
 
 			channelCap = suite.chainA.GetChannelCapability(path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
 		}, true},
+		{"success on channel in tryupgrade flush status in flushing", func() {
+			// setup uses an UNORDERED channel
+			suite.coordinator.Setup(path)
+
+			// create packet commitment
+			sequence, err := path.EndpointA.SendPacket(defaultTimeoutHeight, disabledTimeoutTimestamp, ibctesting.MockPacketData)
+			suite.Require().NoError(err)
+
+			// create packet receipt and acknowledgement
+			packet = types.NewPacket(ibctesting.MockPacketData, sequence, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, defaultTimeoutHeight, disabledTimeoutTimestamp)
+			err = path.EndpointB.RecvPacket(packet)
```

Comment:
super duper nit: 
```suggestion
			suite.Require().NoError(path.EndpointB.RecvPacket(packet))
```

Reply:
I am honestly not so opinionated, it's just some feedback I've gotten before that I thought made sense so I'm passing it along now. but I also see your reasoning with the line length, so I think it's just up to you here :) no need to push another commit if you feel it's more readable this way!